CI fails to build main documentation

CI fails to build main documentation: https://github.com/huggingface/smolagents/actions/runs/17161891163/job/48692957832
```python
Building docs for smolagents ../smolagents/docs/source/ko ../build_dir/smolagents/main/ko

Building the MDX files:   0%|   
Building the MDX files: 100%|██████████| 8/8 [00:00<00:00, 2406.89it/s]
Traceback (most recent call last):
  File "/home/runner/work/smolagents/smolagents/.venv/bin/doc-builder", line 10, in <module>
    sys.exit(main())
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/doc_builder/commands/doc_builder_cli.py", line 49, in main
    args.func(args)
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/doc_builder/commands/build.py", line 102, in build_command
    build_doc(
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/doc_builder/build_doc.py", line 371, in build_doc
    sphinx_refs = check_toc_integrity(doc_folder, output_dir)
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/doc_builder/build_doc.py", line 449, in check_toc_integrity
    toc = yaml.safe_load(f.read())
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/composer.py", line 82, in compose_node
    node = self.compose_sequence_node(anchor)
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/composer.py", line 111, in compose_sequence_node
    node.value.append(self.compose_node(node, index))
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/composer.py", line 82, in compose_node
    node = self.compose_sequence_node(anchor)
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/composer.py", line 111, in compose_sequence_node
    node.value.append(self.compose_node(node, index))
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/scanner.py", line 223, in fetch_more_tokens
    return self.fetch_value()
  File "/home/runner/work/smolagents/smolagents/.venv/lib/python3.10/site-packages/yaml/scanner.py", line 577, in fetch_value
    raise ScannerError(None, None,
yaml.scanner.ScannerError: mapping values are not allowed here
  in "<unicode string>", line 18, column 36:
        title: (번역중) "Human-in-the-Loop: Customize agent plan interacti ... 
                                       ^
Error: Process completed with exit code 1.
```

It seems the issue was introduced by:
- #1644

--- a/docs/source/ko/_toctree.yml
+++ b/docs/source/ko/_toctree.yml
@@ -15,7 +15,7 @@
   - local: in_translation
     title: (번역중) Using different models
   - local: in_translation
-    title: (번역중) "Human-in-the-Loop: Customize agent plan interactively"
+    title: "(번역중) Human-in-the-Loop: Customize agent plan interactively"
   - local: in_translation
     title: (번역중) Async Applications with Agents
 - title: Reference

